src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2800,15 +2800,16 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 2 failures in `PowerShell-CI-macos`
Tests for (error, warning, etc) action preference.Setting ErrorActionPreference to stop prevents user from getting the error exception.$err.Count
```powershell
Expected 1, but got 2.
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Language/Scripting/ActionPreference.Tests.ps1: line 79
79:         It '$err.Count' { $err.Count | Should -Be 1 }
```
Function Pipeline Behaviour.Output Behaviour on Error States.still completes Cleanup {} execution when StopProcessing() is triggered mid-Cleanup {}
```powershell
Expected exactly 'CLEANUP', but got $null.
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Language/Scripting/PipelineBehaviour.Tests.ps1: line 119
119:             $powershell.Streams.Information[0].MessageData | Should -BeExactly "CLEANUP"
```